### DISCUSSION TITLE
Check for existence of mark in extension

### DISCUSSION THREAD

**Discussion by dipanshubhanot** [ueberdosis/tiptap#2243] [General Questions] [answered]
I want to make sure that a particular mark extension(A) excludes another mark(B). However, there is a runtime `UnknownMarkError` if mark B is not added to the editor and mark A excludes mark B. 

Is there a way for mark A to check if mark B exists to avoid this runtime error?

**Comment by philippkuehn** [comment, marked as answer]
Unfortunately not. This is a limitation of ProseMirror.
